pkg/driver/postgres/postgres.go

```diff
@@ -115,10 +124,12 @@ func (drv *Driver) openPostgresDB() (*sql.DB, error) {
 		return nil, err
 	}
 
-	// connect to postgres database
-	postgresURL.Path = "postgres"
+	// connect to postgres database, unless this is a Redshift connection
+	if drv.databaseURL.Scheme != "redshift" {
+		postgresURL.Path = "postgres"
+	}
 
-	return sql.Open("postgres", postgresURL.String())
+	return sql.Open("postgres", connectionString(postgresURL))
```

Comment:
why is this necessary? we already ran it through `connectionString()` above

Reply:
Good catch, not sure why I thought this was necessary.